tests/go.sum

```diff
@@ -129,6 +129,8 @@ github.com/kyma-incubator/compass/components/connector v0.0.0-20230607082115-977
 github.com/kyma-incubator/compass/components/connector v0.0.0-20230607082115-977757624f7e/go.mod h1:K740+tPkGSt9EDSesTwrtHPfRHAiFnj/wJyYaLvuDYU=
 github.com/kyma-incubator/compass/components/director v0.0.0-20230607082115-977757624f7e h1:YOi7R2EBIfYmp+7645bFxE0f0sEJzL/q8HDuPhw/Yik=
 github.com/kyma-incubator/compass/components/director v0.0.0-20230607082115-977757624f7e/go.mod h1:gbMcV8XF0oQOtlACKrF4B9OFAAFEkUBd8yEYKJUtP7M=
```

Comment:
go mod tidy!!!

Reply:
I'll open a follow-up PR to add the missing unit tests and a few more adaptations and there I'll run `go mod tidy`. Thanks for noticing it!